tigerpath/majors_and_certificates/majors/POL_2018.json

```
@@ -15,197 +15,431 @@
       "email": "apatten@princeton.edu"
     },
     {
-      "type": "Departmental Representative",
-      "name": "Keren Yarhi-Milo",
-      "email": "kyarhi@princeton.edu"
+      "type": "Director of Undergraduate Studies",
+      "name": "John Kastellec",
+      "email": "jkastell@princeton.edu"
     }
   ],
   "req_list": [
     {
       "name": "Prerequisites",
       "max_counted": 1,
       "min_needed": 2,
-      "explanation": "Normally, students entering the department must have successfully completed at least two courses offered by the Department of Politics on a graded basis before the end of their sophomore year.  It is strongly recommended that one or both of the prerequisites be at the 200-level. Courses taken as prerequisites will be counted as departmentals.",
+      "explanation": "Normally, students entering the department must have successfully completed at least two courses offered by the Department of Politics on a graded basis before the end of their sophomore year. It is strongly recommended that one or both of the prerequisites be at the 200-level. Courses taken as prerequisites will be counted as departmentals.",
       "completed_by_semester": 4,
       "double_counting_allowed": true,
       "course_list": [
         "POL *"
       ]
     },
     {
-      "name": "Departmentals",
+      "name": "Analytical Requirement",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": "Concentrators are required to take a Politics course in systematic analysis, normally no later than the first semester of the junior year. The Department strongly recommends completing this requirement before the end of the sophomore year. The analytical course must be taken for a grade - not P/D/F.\n\nThe analytical requirement can be met by taking a Politics course in quantitative methods (POL 345/SOC 305/WWS 211 or POL 346), formal theory (POL 250 or POL 347/ECO 347), or experimental methods (POL 341).\n\nThe course used to fulfill the analytical requirement cannot be used to satisfy primary, secondary or third field requirements, but it does count as a departmental.",
+      "completed_by_semester": 7,

```

Why 7? They're saying Junior Fall at the latest, which would be 5, right? They're even strongly recommending 4th semester!
```suggestion
      "completed_by_semester": 5,
```